sdk/edgeorder/Azure.ResourceManager.EdgeOrder/api/Azure.ResourceManager.EdgeOrder.net8.0.cs

```diff
@@ -1185,6 +1549,64 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public static bool operator !=(Azure.ResourceManager.EdgeOrder.Models.ProductWeightMeasurementUnit left, Azure.ResourceManager.EdgeOrder.Models.ProductWeightMeasurementUnit right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class ProvisioningDetails : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.EdgeOrder.Models.ProvisioningDetails>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.EdgeOrder.Models.ProvisioningDetails>
```

Comment:
let us polish the name of this type

Reply:
Renamed to `EdgeOrderProvisioningDetails`